app/scripts/modules/google/serverGroup/configure/serverGroupConfiguration.service.js

```diff
@@ -371,6 +383,17 @@ module.exports = angular.module('spinnaker.serverGroup.configure.gce.configurati
       });
     }
 
+    function refreshHttpHealthChecks(command, skipCommandReconfiguration) {
+      return cacheInitializer.refreshCache('httpHealthChecks').then(function() {
```

Comment:
This would be a little more readable if the promises weren't nested, e.g.:
`cacheInitializer.refreshCache('httpHealthChecks')
  .then(function() {
     return gceHttpHealthCheckReader.listHttpHealthChecks();
  })
  .then(function(httpHealthChecks) { ... })`